libs/utils/decorate.js

```diff
@@ -0,0 +1,129 @@
+
+import { decorateLinkAnalytics } from './analytics.js';
+
+export function decorateButtons(el) {
+  const buttons = el.querySelectorAll('em a, strong a');
+  if (buttons.length === 0) return;
+  buttons.forEach((button) => {
+    const parent = button.parentElement;
+    const buttonType = parent.nodeName === 'STRONG' ? 'blue' : 'outline';
+    button.classList.add('con-button', buttonType);
+    parent.insertAdjacentElement('afterend', button);
+    parent.remove();
+  });
+  const actionArea = buttons[0].closest('p');
+  actionArea.classList.add('action-area');
+  actionArea.nextElementSibling?.classList.add('supplemental-text', 'body-XL');
+}
+
+export function initIcons(el) {
+  // eslint-disable-next-line no-useless-escape
+  const regex = /\{\{icon-(.*?)\}\}+/g; // {{icon-*}}
+  const finds = el.textContent.match(regex);
+  finds?.forEach((find) => {
+    const str = find.replace('{{', '').replace('}}', '');
+    el.innerHTML = el.innerHTML.replace(`{{${str}}}`, `<span class="icon">${str}</span>`);
+  });
+  const icons = el.querySelectorAll('.icon');
+  icons.forEach((icon) => {
+    icon.parentElement.classList.add('icon-area');
+    if (icon.textContent.includes('persona')) {
+      icon.parentElement.classList.add('persona-area');
+    }
+  });
+}
+
+export async function formatIconLibraryData(json) {
+  let library = {};
+  json['icons']?.data.forEach((icon) => {
+    const iconValues = {};
+    Object.entries(icon).forEach((value) => {
+      const iconValue = value[1];
+      if (iconValue) {
+        iconValues[value[0]] = iconValue;
+      }
+    });
```

Comment:
nit: destructure object entries

```suggestion
    Object.entries(icon).forEach(([key, iconValue]) => {
      if (iconValue) iconValues[key] = iconValue;
    });
```
or you could use reduce:
```
const iconValues = [...Object.entries(icon)].reduce((values, [key, val]) => {
  if (val) values[key] = val;
  return values;
}, {});
```
Though I'm not sure that's actually better in terms of readability.